Assets/LeapMotion/Resources/LeapCG.cginc

```diff
@@ -89,6 +97,22 @@ float4 LeapRawColorBrightnessUV(float2 uv){
 	#endif
 }
 
+float4 WarpScreenPosition(float4 sp) {
```

Comment:
Probably should be in a separate PR after we get this merged in, but it probably would be a nice idea to roll the warping into the existing methods instead of creating a new ones.  We could then control whether or not timewarp was used via shader variants, in the same way that variants are currently used to differentiate between color / ir passthrough.  That way we can hide all the underlying complexity, and also prevent the need for writing multipler versions of all our shaders.